onnxruntime/core/framework/session_state.cc

```diff
@@ -788,8 +788,6 @@ Status SessionState::FinalizeSessionState(const std::basic_string<PATH_CHAR_TYPE
   } else {
 #if !defined(ORT_MINIMAL_BUILD)
     ORT_RETURN_IF_ERROR(PopulateKernelCreateInfo(kernel_registry_manager));
-    return FinalizeSessionStateImpl(graph_location, kernel_registry_manager, nullptr, session_options,
```

Comment:
Not needed anymore ?

Reply:
Accidental duplicate from merging. It's called a few lines down outside of this 'if'